gene_benchmark/tests/test_tasks.py

```diff
@@ -405,8 +400,8 @@ def test_entities_task_summary(self):
         assert "sample_size" in summary
 
     def test_list_subtests(self):
-        task_name = "long vs short range TF"
-        tsk = load_task_definition(task_name, tasks_folder=_get_tasks_folder())
+        task_name = "simple_bin"
```

Comment:
is it `symbol_bin` or `simple_bin`?

Reply:
Ok, we had both, but you are right. It is redundant. Removed the simple.